Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,44 @@
+//1. Declare a variable on each type using two options: plain creation, creation with constructor.
+let numberVar = 32;
```

Comment:
Better always use  `const` if you are not reassign variable

Reply:
also you missed creation with constructor

Reply:
thanks! fixed